AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/tier/tier-edit/tier-edit.component.ts

```diff
@@ -49,13 +49,12 @@ export class TierEditComponent {
         this.quotasTableDisplayedColumns = ["select", "metricName", "max", "period"];
         this.editMode = false;
         this.loading = true;
-        this.tier = {} as Tier;
         this.disabled = false;
         this.tier = {
             id: "",
             name: "",
             quotas: [],
-            numberOfIdentities: 0
+            deleteable: true
```

Comment:
Should be `false` by default. I prefer that things are not deletable when they should be, rather than the other way around.